stdlib/warnings.pyi

```diff
@@ -61,6 +61,7 @@ class catch_warnings:
     def __new__(cls, *, record: Literal[True], module: Optional[ModuleType] = ...) -> _catch_warnings_with_records: ...
     @overload
     def __new__(cls, *, record: bool, module: Optional[ModuleType] = ...) -> catch_warnings: ...
+    def __init__(self, *, record: bool = ..., module: Optional[ModuleType] = ...) -> None: ...
```

Comment:
From the mypy_primer output it looks like adding this method breaks the `__new__` overloads. Maybe we can achieve the desired effect by overloading `__init__` instead?

Reply:
I'll check it out later, worst cast, I'll remove the \_\_init__